pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
in general I have no strong opinion here but what was main reason to drop gradle completely?

Reply:
Good question, I have only light preference towards maven. 

I had to update the build to make LD_LIBRARY_PATH usage automatic, and I was thinking, if it's a polite thing or not to switch back to maven it, but this repo was not updated recently I thought It can be changed if it makes faster to improve the build.

But let me share my experiences with gradle:

When Gradle was introduced, I started to use it as I really liked the more flexible model (compared to maven) and I thought eventually it should have wide adoption. But over the time I switched back to maven:

 1. The flexible artifact tagging (based on IvY) was not possible as everything was in maven repo format anyway. Meantime, I suffered from small incompatibilities between maven and IvY and maven download speed was also greatly improved
 2. While I liked the idea to define the build code in DSL it turned out to be more complex especially for community projects. Groovy is not as well known as XML.. And not as type-safe, many parameters could be found only from the source code (if the doc is not updated). At the end, it was significant harder to understand the complex build processes. 
 3. Uploading artifacts to maven repository was easier with maven (as it was tested more with maven) 
 4. Tooling was always like 95% of maven (including IDE and artifact uploading.)
 
 Gradle was widely adopted only by the android projects AFAIK. I see the advantages of Gradle only in big projects with many subprojects, where simplicity of maven can be a limiting factor.

 In our case, I think maven is better:

 1. Maven is more well-known, it can be easier for the community to understand the changes
 2. I would like to upload artifacts to maven central / ghcr, and maven has better support for it. 
 3. Maven is an Apache project (--> well-defined project governance)   
 
 
But again, it's not a strong preference, either. For a more active project (or for android) I wouldn't change it, I am pretty sure  that everything can be done in gradle (just more debugging for me ;-) )
 
I am also thinking to put gradle descriptor to the example project to provide both option for users.

Reply:
Sorry for delayed response. Somehow I missed mail with notification.

Thank for this interesting comparison! As I said earlier I had no objections or so but nice to have such a comprehensive answer :)